src/docfx/docset/Docset.cs

```diff
@@ -299,6 +290,20 @@ private HashSet<Document> CreateBuildScope(IEnumerable<Document> redirections)
             }
         }
 
+        private TemplateEngine CreateTemplate()
+        {
+            if (string.IsNullOrEmpty(Config.Theme))
+            {
+                return new TemplateEngine(Path.Combine(AppContext.BaseDirectory, "theme"), Locale);
```

Comment:
what's that?

Reply:
this will throw undefined exception for docs repos, so just return null before we have real default theme?